MiniApp.podspec

```diff
@@ -12,14 +12,20 @@ Pod::Spec.new do |s|
   s.authors      = "Rakuten Ecosystem Mobile"
   s.summary      = "Rakuten's Mini App SDK"
   s.homepage     = "https://github.com/rakutentech/ios-miniapp"
-  s.license      = { :type => 'MIT', :file => 'LICENSE' }
-  s.source       = { :git => "https://github.com/rakutentech/ios-miniapp.git", :tag => 'v' + s.version.to_s }
+  s.license      = {
+    :type => 'MIT',
+    :file => 'LICENSE'
+  }
+  s.source       = {
+    :git => "https://github.com/rakutentech/ios-miniapp.git",
+    :tag => 'v' + s.version.to_s,
```

Comment:
Shouldn't you add `:submodules => true `? Because otherwise the the submodule won't get downloaded when a client uses this pod, right?

Reply:
yes sorry, I appeared that when I tested locally I had no need for this parameter. Hopefully I tested again by using this line into the podfile : `pod 'MiniApp', git: 'https://github.com/Climbatize/ios-miniapp', branch: 'master', submodules: true` and it appeared this parameter was important. Thanks for making me double check